src/libraries/Microsoft.PowerFx.Interpreter/Microsoft.PowerFx.Interpreter.csproj

```diff
@@ -32,6 +32,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="System.Drawing.Common" Version="6.0.0" />
```

Comment:
> <PackageReference Include="System.Drawing.Common" Version="6.0.0" /> [](http://example.com/codeflow?start=3&length=69)

We need to be wary of package references...

Is this just for System.Drawing.ColorTranslator?

Reply:
Removed